src/tokens.ts

```diff
@@ -15,6 +15,8 @@ export const PLUGIN_IDS = {
   systemPrompts: '@jupyterlite/ai:system-prompts'
 };
 
+export type modelUsage = 'chat' | 'completer';
```

Comment:
Maybe this type can also start with an uppercase letter, for consistency with the JupyterLab code base?

```suggestion
export type ModelUsage = 'chat' | 'completer';
```